twitter_analysis/app.py

```diff
@@ -0,0 +1,166 @@
+from flask import Flask, render_template
+import os
+from pymongo import MongoClient
+import requests
+import csv
+from decouple import config
+from datetime import datetime
+import pickle
+from xgboost import XGBClassifier
+import numpy as np
+from collections import deque
+import pandas as pd 
+from pandas import MultiIndex, Int16Dtype
+import json
+app = Flask(__name__)
+
+
+class TweetCollection:
+    def __init__(self):
+        f = open('topics.json')
+        self.query_list = deque(json.load(f))
+
+    def get_tweets(self):
+        while True:
+            self.get_all_tweets()
+
+    def get_all_tweets(self):
```

Comment:
This function is entirely too long and cluttered. Work into breaking this into smaller more digestable chucks.